fms_core/serializers.py

```diff
@@ -43,6 +53,41 @@ class Meta:
         fields = "__all__"
 
 
+class SampleExportSerializer(serializers.ModelSerializer):
```

Comment:
When I try retrieving samples at  `/api/samples/list_export/`
I get the following error:
```
TypeError at /api/samples/list_export/
Object of type Individual is not JSON serializable
```

Reply:
Hi @zxenia, I just merged the changes Ulysse made on master to this branch. I tested it with the latest db and it is fine on my machine, let me know if you still get an error.

Reply:
I fixed the error that I get locally with the following changes in SampleExportSerializer's methods:
added `label`

```
def get_father_id(self, obj):
    father = '' if obj.individual.father is None else obj.individual.father.label
    return father

def get_mother_id(self, obj):
    mother = '' if obj.individual.mother is None else obj.individual.mother.label
    return mother
```

Reply:
I would expect get_father_id to return the id ... not the label... I think the desired info is the label ... maybe rename the function ...